docs/sources/developers/plugins/share-a-plugin.md

```diff
@@ -1,120 +1,69 @@
 +++
-title = "Share a plugin"
+title = "Package a plugin"
 type = "docs"
+aliases = ["/docs/grafana/latest/developers/plugins/share-a-plugin/"]
 +++
 
-# Share a plugin
+# Packaging plugins
```

Comment:
Maybe Package a plugin or Package plugins? That would match the title.

Reply:
I was thinking that since the task heading in this document is "Package the plugin", I didn't want to repeat myself. It does include a concept topic as well so I went with a gerund here. WDYT?